tests/testthat/test-utils.R

```diff
@@ -1,15 +1,15 @@
 
-test_that("util_replace_null returns NA", {
+test_that("`util_replace_null` returns NA", {
   expect_equal(util_replace_null(NULL), NA)
   expect_equal(util_replace_null(1), 1)
 })
 
-test_that("util_df_to_list is working", {
+test_that("`util_df_to_list` is working", {
   expect_type(util_df_to_list(ggplot2::mpg), "list")
 })
 
 # There's no test for util_list_to_df1, should we add one?
-test_that("util_list_to_df2 is working", {
+test_that("`util_list_to_df2` is working", {
   expect_equal(util_list_to_df2(NULL), NA)
   expect_equal(util_list_to_df2(NA), NA)
 
```

Comment:
We should add a couple more tests, .e.g to show that the function produces output in desired format

`traits.build::util_df_to_list(dplyr::band_instruments)`